decentraland/unity-renderer
# feat: friends lazy loading

## What does this PR change?

Implementation of lazy loading of friend lists.

## How to test the changes?

1. Open the friends window
2. In case the service is down, right click over `FriendsController` component at `Main` object -> Force Initialization
3. Open the friend list tab, mocked friends should be displayed
4. Use the search bar to look for any friend, mocked friend entries should be matched
5. In case you have more friends to load, click on the show more button
6. Open the friend request tab, send/received requests should be displayed
7. In case you have more requests to load, click on the show more button

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
